example/anonymous-off-chain/voting.ts

```diff
@@ -146,6 +146,17 @@ export async function submitVotes(processId: string, processParams: ProcessState
     const verifyProof = await ZkSnarks.verifyProof(JSON.parse(Buffer.from(vKey).toString()), zkProof.publicSignals as any, zkProof.proof as any)
     if (config.stopOnError) assert(verifyProof)
 
+    // the next line is to place the 'nullifier' into the position 0 of
+    // publicSignals array. This is because in the Vochain, the rest of
+    // publicSignals from the array are not going to be used, as those inputs
+    // are going to be computed from known info of the process by the Vochain
+    // itself. And the Vochain only needs the 'nullifier' being in the first
+    // position of the array. In fact, the array could contain only 1 element
+    // (the 'nullifier').
+    zkProof.publicSignals = [nullifier];
```

Comment:
Have you tested this directly in JS to make it work, right? Just asking because with TS it shouldn't compile. The type definition of  `zkProof.publicSignals` is an array of strings, not from bigint (which is the type of the `nullifier`)

Reply:
I've tested it with `npm start`. But added the comment with the explanation on what content is expected to be in the `publicSignals` array, so you can update it in the more suitable way, so if there is a better way update it without any doubt ^^